### PR TITLE
[new] [author] Added zuix.js library

### PR DESCRIPTION
**Pull request by genemars**
# Pull request template for new library

Pull request for issue: #13339
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/zuixjs/zuix
* Official website (optional, not the repository): https://zuixjs.org
* NPM package url (optional): zuix-dist
* License and its reference: Apache-2.0
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 10
  * Count of stars: 57
  * Count of forks: 5
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week: 158
  * Downloads in the last month:

## Essential checklist

* [X] I'm the author of this library
* [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [X] This lib was not found on cdnjs repo
* [X] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
* [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
* [ ] More than 800 downloads stats per month on npm registry
* [X] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [X] Has valid tags for each versions (for git auto-update)
* [ ] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [X] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [X] The parent of the commit(s) in the PR is not older than 3 days.
* [X] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [X] Close corresponding issue in commit message
* [X] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by MattIPv4** [comment]
Resolves #13339

File: ajax/libs/zuix/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zuixjs/zuix.git"
+  },
+  "npmName": "zuix-dist",
+  "npmFileMap": [
+    {
+      "basePath": "js",
+      "files": [
+        "zuix?(.min).+(js|map)"
```
Comment (zackbloom) [inline ajax/libs/zuix/package.json]: Is this a regex? Should the `?` appear after the `(.min)`?

Comment (MattIPv4) [reply ajax/libs/zuix/package.json]: These are globs iirc, not regex.

Comment (MattIPv4) [reply ajax/libs/zuix/package.json]: image
http://www.globtester.com/#p=eJyrKs2ssNfQy83M09TT1sgqrslNLNAEAE%2BABxQ%3D&r=eJyzScksU0jOSSwutlXKTSxJzlCyqyrNrNDLKrbRB0rZ2eCQz83MI0pNbmIBpiKofElqcYlNUpEdNgUIcQApKTg4&

Comment (zackbloom) [reply ajax/libs/zuix/package.json]: This is awesome, thank you for showing it to me @MattIPv4

**Comment by MattIPv4** [comment]
Hi there!

This library doesn't have enough popularity (like more than 200 stars on GitHub or 800 downloads per month on npm). Please take a look at our [pull request template, essentials checklist](https://github.com/cdnjs/cdnjs/blob/master/PULL_REQUEST_TEMPLATE.md#essential-checklist) for more information regarding this. 📖

Please open an issue again when the library has more popularity, so that we can process it properly and add the library to cdnjs! Thanks for your great work and contribution, sorry we couldn't add your request at this time. 😄

**Pull request closed without merging**